README.md

```diff
@@ -52,27 +52,82 @@ the provided `OpenTracing::Tracer`
 If you prefer direct control to singletons, manage ownership of the
 `Tracer` implementation explicitly.
 
-### Starting an empty trace by creating a "root span"
+### Scopes and within-process propagation
 
-It's always possible to create a "root" `Span` with no parent or other causal
-reference.
+For any thread, at most one `Span` may be "active". Of course there may be many
+other `Spans` involved with the thread which are (a) started, (b) not finished,
+and yet (c) not "active": perhaps they are waiting for I/O, blocked on a child
+`Span`, or otherwise off of the critical path.
+
+It's inconvenient to pass an active `Span` from function to function manually,
+so OpenTracing requires that every `Tracer` contains a `ScopeManager` that
+grants access to the active `Span` through a `Scope`. Any `Span` may be
+transferred to another callback or thread, but not `Scope`.
+
+#### Accessing the active Span through Scope
 
 ```ruby
-span = OpenTracing.start_span("operation_name")
-span.finish
+# Access to the active span is straightforward.
+scope = OpenTracing.scope_manager.active
```

Comment:
I wonder if we should also add conventient method to access span directly like https://github.com/opentracing/opentracing-python/blob/v2.0.0/opentracing/tracer.py#L55-L63. Basically `OpenTracing.active_span`. This looks a lot nicer than trying to get it through scope_manager + scope.

Reply:
Excellent suggestion. I have added this in #30.